hexrd/instrument/detector.py

```diff
@@ -91,7 +91,6 @@ def cart_to_angles(self, xy_data, rmat_s=None, tvec_s=None,
             DESCRIPTION.
 
         """
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.